i18n/middleware.py

```diff
@@ -0,0 +1,36 @@
+import re
+import warnings
+
+from django.conf import settings
+from django.utils import translation
+from django.http import HttpResponseRedirect
```

Comment:
is this used? or `warnings` above?

Reply:
oh oops. It was at one point in my development but not anymore. I'll remove